var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -16,6 +15,7 @@ class Mercury(CMakePackage):
     maintainers("soumagne")
     tags = ["e4s"]
     version("master", branch="master", submodules=True)
+    version("2.3.0", sha256="e9e62ce1bb2fd482f0e85ad75fa255d9750c6fed50ba441a03de93b3b8eae742")
```

Comment:
```suggestion
    maintainers("soumagne")
    
    tags = ["e4s"]
    
    version("master", branch="master", submodules=True)
    version("2.3.0", sha256="e9e62ce1bb2fd482f0e85ad75fa255d9750c6fed50ba441a03de93b3b8eae742")
```

@soumagne thanks for the PR! This isn't something you touched but just to bring the package up in line with the common styling of others can we add a newline between the maintainers, tags, and versions?

Reply:
ok done

Reply:
Great! Thank you!